### PR TITLE
finanzamtmeldung faq, status cloud tse update

### PR DESCRIPTION
**Pull request by aedler-ft**

### CODE REVIEW

File: qna/DE-Finanzamtmeldung.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Wann muss ich die Verwendung einer TSE dem Finanzamt melden?
+
+## Question
+
+Welche Informationen muß ich dem Finanzamt übermitteln?
+
+## Question
+
+Ist im Falle eines TSE-Verlusts oder -Diebstahls eine Meldung und Sperrung der TSE, z. B.im Portal oder ggü. dem Finanzamt notwendig?
```
Comment (magdignomad) [inline qna/DE-Finanzamtmeldung.md:11]: z. B.[ ]im

File: qna/DE-Finanzamtmeldung.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Metadata tags
+
+lang-de, market-de, middleware, PosCreator, PosDealer, Consultant
+
+## Answer
+
+Von der Mitteilung (Meldung der TSE an das Finanzamt, Anm.) nach § 146a Absatz 4 AO ist bis zum Einsatz einer elektronischen Übermittlungsmöglichkeit abzusehen. Der Zeitpunkt des Einsatzes der elektronischen Übermittlungsmöglichkeit wird im Bundessteuerblatt Teil I gesondert bekannt gegeben.
```
Comment (magdignomad) [inline qna/DE-Finanzamtmeldung.md:19]: Perhaps we can do a link to the target of this quote?

File: qna/DE-Finanzamtmeldung.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+Es sind aktuell daher weder der Zeitpunkt noch der Inhalt der zu übermittelnden Informationen bis zur oben beschriebenen Veröffentlichung der elektronischen Übermittlungsmöglichkeit bekannt. 
+
+Im Falle eines Verlustes bzw. Diebstahls einer TSE empfehlen wir den Vorfall im Rahmen der Verfahrensdokumentation zu dokumentieren, eine Abmeldung der TSE beim Finanzamt vorzunehmen (falls elektronische Übermittlungsmöglichkeiten vorhanden sind, siehe oben), eine neue TSE zu beschaffen und diese im Rahmen des üblichen Verfahrens der Beschaffung einer neuen TSE mit einer neuen Cashbox-Konfiguration inkl. neuer Queue in Betrieb zu nehmen. Eine gesonderte Sperrung im Portal oder beim Finanzamt ist hierbei nicht notwendig.
```
Comment (magdignomad) [inline qna/DE-Finanzamtmeldung.md:23]:
```suggestion
Im Falle eines Verlustes bzw. Diebstahls einer TSE empfehlen wir den Vorfall im Rahmen der Verfahrensdokumentation zu dokumentieren, eine Abmeldung der TSE beim Finanzamt vorzunehmen (falls elektronische Übermittlungsmöglichkeiten vorhanden sind, siehe oben), eine neue TSE zu beschaffen und diese im Rahmen des üblichen Verfahrens der Beschaffung mit einer neuen Cashbox-Konfiguration inkl. neuer Queue in Betrieb zu nehmen. Eine gesonderte Sperrung im Portal oder beim Finanzamt ist hierbei nicht notwendig.
```

File: qna/DE-Finanzamtmeldung.md (hunk 1 above)
Comment (aedler-ft) [reply qna/DE-Finanzamtmeldung.md:11]: fixed

**Pull request merged**